wp-seo-main.php

```diff
@@ -342,7 +342,15 @@ function wpseo_init() {
 	WPSEO_Meta::init();
 
 	if ( version_compare( WPSEO_Options::get( 'version', 1 ), WPSEO_VERSION, '<' ) ) {
```

Comment:
I read this as:

_On `plugins_loaded`, when the version stored in the database is lower than the version retrieved from the file constant, do opcache invalidation._

This begs several questions:

1. If the file constant `WPSEO_VERSION` was already updated, wouldn't that mean that there's no need for an opcache reset, as we're already seeing the updated files, and not the old ones? If the opcache would not have been properly updated after downloading the files, then `WPSEO_VERSION` would still be at the old version, wouldn't it?

2. If the database update is not immediate, wouldn't that mean that this opcache reset can be triggered multiple times until the database retrieves the new value? If the site is being hit by a thousand requests right after the update, these thousand requests would trigger the opcache invalidation a thousand time in parallel, from what I can see.